Recipes/Controller/RecipeDetailViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  RecipeDetailViewController.swift
+//  Recipes
+//
+//  Created by krikaz on 3/12/20.
+//  Copyright © 2020 Lambda Inc. All rights reserved.
+//
+
+import UIKit
+
+class RecipeDetailViewController: UIViewController {
+    
+    @IBOutlet weak var recipeLabel: UILabel!
+    @IBOutlet weak var recipeDescription: UITextView!
```

Comment:
A good habit for UI properties is to suffix it with the type like you did for `recipeLabel`. You could do `recipeDescriptionTextView`.